pkg/exporter/process.go

```diff
@@ -172,17 +181,26 @@ func InitExportingProcess(input ExporterInput) (*ExportingProcess, error) {
 		} else if input.CollectorProtocol == "udp" { // use DTLS
 			// TODO: support client authentication
 			if len(tlsConfig.CertData) > 0 || len(tlsConfig.KeyData) > 0 {
-				klog.Error("Client-authentication is not supported yet for DTLS, cert and key data will be ignored")
+				return nil, fmt.Errorf("client-authentication is not supported yet for DTLS")
+			}
+			if tlsConfig.MinVersion != 0 && tlsConfig.MinVersion != tls.VersionTLS12 {
```

Comment:
I think the tls version would be decided by server? Do we need to specify the minTlsVersion for client? Or it's not harmful to set it on both side?